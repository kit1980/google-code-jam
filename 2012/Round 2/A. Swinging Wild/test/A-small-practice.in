30
3
3 4
4 10
6 10
9
3
3 4
4 10
7 10
9
2
6 6
10 3
13
2
6 6
10 3
14
1
10 25
25
99
798169 1000000
1596304 798132
1596306 798131
1596308 798130
1596310 798129
1596312 798128
1596314 798127
1596316 798126
1596318 798125
1596320 798124
1596322 798123
1596324 798122
1596326 798121
1596328 798120
1596330 798119
1596332 798118
1596334 798117
1596336 798116
1596338 798115
2394438 798130
3192568 798130
3990664 798093
3990666 798092
3990668 798091
3990670 798090
3990672 798089
3990674 798088
3990676 798087
3990678 798086
3990680 798085
3990682 798084
3990684 798083
3990686 798082
3990688 798081
3990690 798080
3990692 798079
3990694 798078
3990696 798077
3990698 798076
4788771 798078
5586850 798079
6384895 798042
6384897 798041
6384899 798040
6384901 798039
6384903 798038
6384905 798037
6384907 798036
6384909 798035
6384911 798034
6384913 798033
6384915 798032
6384917 798031
6384919 798030
6384921 798029
6384923 798028
6384925 798027
6384927 798026
6384929 798025
7182951 798028
7980979 798028
8778973 797991
8778975 797990
8778977 797989
8778979 797988
8778981 797987
8778983 797986
8778985 797985
8778987 797984
8778989 797983
8778991 797982
8778993 797981
8778995 797980
8778997 797979
8778999 797978
8779001 797977
8779003 797976
8779005 797975
8779007 797974
9576981 797974
10374955 797974
11172895 797937
11172897 797936
11172899 797935
11172901 797934
11172903 797933
11172905 797932
11172907 797931
11172909 797930
11172911 797929
11172913 797928
11172915 797927
11172917 797926
11172919 797925
11172921 797924
11172923 797923
11172925 797922
11172927 797921
11172929 797920
11970849
99
91111 1000000
111610 877958
114570 966884
114624 706104
121944 555508
122699 814565
129045 822179
149503 598744
150668 810599
161674 647316
182206 91092
182208 91091
182210 91090
182212 91089
182214 91088
182216 91087
182218 91086
182220 91085
182222 91084
273305 91085
364390 91085
373415 668103
389788 942539
394894 770538
396716 523668
403261 798519
410918 562567
425046 887250
437865 743800
438539 600619
512672 74130
512674 74129
512676 74128
512678 74127
512680 74126
512682 74125
512684 74124
512686 74123
512688 74122
586808 74124
660932 74124
667548 996029
674506 973095
694742 678105
698050 741323
715508 879266
717408 922192
723094 516094
724910 910595
729921 706987
798894 68970
798896 68969
798898 68968
798900 68967
798902 68966
798904 68965
798906 68964
798908 68963
798910 68962
867866 68968
936834 68968
941528 621664
943456 932372
951946 838962
958631 736794
972515 925170
990358 836580
998757 865557
999560 873821
1003883 979460
1005786 68949
1005788 68948
1005790 68947
1005792 68946
1005794 68945
1005796 68944
1005798 68943
1005800 68942
1005802 68941
1074735 68949
1143684 68949
1159128 665278
1166690 641780
1176067 626411
1179177 562657
1184329 629178
1196731 613584
1196849 668448
1197767 987757
1201573 587907
1212617 68930
1212619 68929
1212621 68928
1212623 68927
1212625 68926
1212627 68925
1212629 68924
1212631 68923
1212633 68922
1281555
5
100 100
105 5
110 5
112 2
114 2
120
99
652358 1000000
1304702 652341
1304704 652340
1304706 652339
1304708 652338
1304710 652337
1304712 652336
1304714 652335
1304716 652334
1957050 652334
2609384 652334
3261704 652317
3261706 652316
3261708 652315
3261710 652314
3261712 652313
3261714 652312
3261716 652311
3261718 652310
3914028 652309
4566338 652310
5218634 652293
5218636 652292
5218638 652291
5218640 652290
5218642 652289
5218644 652288
5218646 652287
5218648 652286
5870928 652292
6523220 652292
7175498 652275
7175500 652274
7175502 652273
7175504 652272
7175506 652271
7175508 652270
7175510 652269
7175512 652268
7827778 652270
8480048 652270
9132304 652253
9132306 652252
9132308 652251
9132310 652250
9132312 652249
9132314 652248
9132316 652247
9132318 652246
9784563 652247
10436810 652247
11089043 652230
11089045 652229
11089047 652228
11089049 652227
11089051 652226
11089053 652225
11089055 652224
11089057 652223
11741273 652230
12393503 652230
13045719 652213
13045721 652212
13045723 652211
13045725 652210
13045727 652209
13045729 652208
13045731 652207
13045733 652206
13697934 652211
14350145 652211
15002342 652194
15002344 652193
15002346 652192
15002348 652191
15002350 652190
15002352 652189
15002354 652188
15002356 652187
15654542 652188
16306730 652188
16958904 652171
16958906 652170
16958908 652169
16958910 652168
16958912 652167
16958914 652166
16958916 652165
16958918 652164
17611082 652164
18263246 652164
18915396 652147
18915398 652146
18915400 652145
18915402 652144
18915404 652143
18915406 652142
18915408 652141
18915410 652140
19567550
99
932554 1000000
1865014 932457
1865016 932456
1865018 932455
1865020 932454
1865022 932453
1865024 932452
1865026 932451
1865028 932450
1865030 932449
1865032 932448
1865034 932447
1865036 932446
1865038 932445
1865040 932444
1865042 932443
1865044 932442
1865046 932441
1865048 932440
1865050 932439
1865052 932438
1865054 932437
1865056 932436
1865058 932435
1865060 932434
1865062 932433
1865064 932432
1865066 932431
1865068 932430
1865070 932429
1865072 932428
1865074 932427
1865076 932426
1865078 932425
1865080 932424
1865082 932423
1865084 932422
1865086 932421
1865088 932420
1865090 932419
1865092 932418
1865094 932417
1865096 932416
1865098 932415
1865100 932414
1865102 932413
1865104 932412
1865106 932411
1865108 932410
2797514 932414
3729928 932414
4662248 932317
4662250 932316
4662252 932315
4662254 932314
4662256 932313
4662258 932312
4662260 932311
4662262 932310
4662264 932309
4662266 932308
4662268 932307
4662270 932306
4662272 932305
4662274 932304
4662276 932303
4662278 932302
4662280 932301
4662282 932300
4662284 932299
4662286 932298
4662288 932297
4662290 932296
4662292 932295
4662294 932294
4662296 932293
4662298 932292
4662300 932291
4662302 932290
4662304 932289
4662306 932288
4662308 932287
4662310 932286
4662312 932285
4662314 932284
4662316 932283
4662318 932282
4662320 932281
4662322 932280
4662324 932279
4662326 932278
4662328 932277
4662330 932276
4662332 932275
4662334 932274
4662336 932273
4662338 932272
4662340 932271
4662342 932270
5594612
1
200000000 300000000
400000000
3
5 5
7 5
9 5
21
2
10 10
11 10
21
7
10 10
15 5
20 9
25 5
30 10
35 5
40 10
45
99
319654 1000000
326072 992840
326729 522969
329395 903993
332033 598946
337116 923157
350373 822066
351384 555527
357151 843846
364114 914498
369796 684434
370726 625203
377993 689995
378481 969048
393754 572549
396848 921675
397924 589488
400733 574818
411923 639247
431769 563359
434221 888318
435247 852389
439235 912449
441256 511896
441542 705497
444365 978572
464393 999970
483274 648901
483630 582598
486732 925951
496635 982216
501447 822168
501895 951683
504354 848425
509086 630062
534397 686697
539863 824199
544764 999991
554925 817634
556397 731471
557235 534427
580193 760047
591122 827749
591511 921072
594620 510648
599004 761010
607785 806068
613340 643349
615541 673965
617856 694693
639212 319555
639214 319554
639216 319553
639218 319552
639220 319551
639222 319550
639224 319549
639226 319548
639228 319547
639230 319546
639232 319545
639234 319544
639236 319543
639238 319542
639240 319541
639242 319540
639244 319539
639246 319538
639248 319537
639250 319536
639252 319535
639254 319534
639256 319533
639258 319532
639260 319531
639262 319530
639264 319529
639266 319528
639268 319527
639270 319526
639272 319525
639274 319524
639276 319523
639278 319522
639280 319521
639282 319520
639284 319519
639286 319518
639288 319517
639290 319516
639292 319515
639294 319514
639296 319513
639298 319512
639300 319511
639302 319510
639304 319509
639306 319508
639308 319507
958815
99
10 62
11 71
13 31
15 99
16 99
17 98
18 58
19 25
20 33
30 9
40 10
41 75
42 63
43 21
44 100
45 33
46 88
47 98
49 54
50 10
60 10
61 85
62 21
63 45
66 20
67 55
68 62
69 89
70 85
80 10
90 10
92 47
93 63
94 80
95 36
96 19
97 20
98 95
99 58
108 9
117 9
118 80
119 14
120 60
122 82
123 61
124 68
125 20
126 67
135 9
144 9
146 56
147 91
148 81
149 89
150 64
151 83
152 44
153 25
162 9
171 9
172 66
173 68
174 36
175 85
176 14
177 47
178 71
179 61
180 9
189 9
190 52
191 44
192 92
193 62
194 22
196 88
197 86
198 97
207 9
216 9
217 81
218 55
219 34
220 40
221 64
223 47
224 73
225 23
234 9
243 9
244 90
245 34
246 69
247 49
248 100
250 97
251 77
252 47
261
99
926542 1000000
1853070 926525
1853072 926524
1853074 926523
1853076 926522
1853078 926521
1853080 926520
1853082 926519
1853084 926518
2779600 926520
3706120 926520
4632626 926503
4632628 926502
4632630 926501
4632632 926500
4632634 926499
4632636 926498
4632638 926497
4632640 926496
5559130 926502
6485632 926502
7412120 926485
7412122 926484
7412124 926483
7412126 926482
7412128 926481
7412130 926480
7412132 926479
7412134 926478
8338605 926485
9265090 926485
10191561 926468
10191563 926467
10191565 926466
10191567 926465
10191569 926464
10191571 926463
10191573 926462
10191575 926461
11118035 926462
12044497 926462
12970945 926445
12970947 926444
12970949 926443
12970951 926442
12970953 926441
12970955 926440
12970957 926439
12970959 926438
13897390 926445
14823835 926445
15750266 926428
15750268 926427
15750270 926426
15750272 926425
15750274 926424
15750276 926423
15750278 926422
15750280 926421
16676696 926426
17603122 926426
18529534 926409
18529536 926408
18529538 926407
18529540 926406
18529542 926405
18529544 926404
18529546 926403
18529548 926402
19455948 926404
20382352 926404
21308742 926387
21308744 926386
21308746 926385
21308748 926384
21308750 926383
21308752 926382
21308754 926381
21308756 926380
22235132 926384
23161516 926384
24087886 926367
24087888 926366
24087890 926365
24087892 926364
24087894 926363
24087896 926362
24087898 926361
24087900 926360
25014259 926361
25940620 926361
26866967 926344
26866969 926343
26866971 926342
26866973 926341
26866975 926340
26866977 926339
26866979 926338
26866981 926337
27793318
95
29834 1000000
30966 793847
35191 518791
41490 590452
52520 735462
54429 929633
54736 967472
56305 574634
59656 29819
59658 29818
59660 29817
59662 29816
59664 29815
59666 29814
59668 29813
89476 29818
119294 29818
122514 878404
126106 887443
129990 709077
133681 541073
138772 581998
140338 609985
147813 643782
149100 29803
149102 29802
149104 29801
149106 29800
149108 29799
149110 29798
149112 29797
178904 29802
208706 29802
209670 580176
214578 991454
216251 665871
216701 655024
225798 517349
228699 533136
236419 682398
238496 29787
238498 29786
238500 29785
238502 29784
238504 29783
238506 29782
238508 29781
268288 29782
298070 29782
302826 673911
308782 826026
311235 742359
314787 832682
318227 620190
322230 651570
327632 856944
327840 29767
327842 29766
327844 29765
327846 29764
327848 29763
327850 29762
327852 29761
357613 29761
387374 29761
390346 677880
395117 533954
396721 605870
401295 707856
402041 751201
404543 661132
411425 869172
435464 24036
435466 24035
435468 24034
435470 24033
435472 24032
435474 24031
435476 24030
459505 24031
483536 24031
484737 656887
491493 709780
492432 642673
493108 927792
495524 983217
496160 892658
503163 764440
522778 19612
522780 19611
522782 19610
522784 19609
522786 19608
522788 19607
522790 19606
542396
99
108814 1000000
124300 963399
129999 804012
137451 514360
140175 730687
141401 739817
142346 819417
143585 988423
147850 885048
156556 641628
157208 745814
162932 986184
165751 808442
167393 659686
168182 587878
170487 923838
178393 752346
184280 633948
184885 941868
200289 937152
201005 782222
202521 681920
202848 700241
206463 980107
208765 506910
217582 108765
217584 108764
217586 108763
217588 108762
217590 108761
217592 108760
217594 108759
217596 108758
217598 108757
217600 108756
217602 108755
217604 108754
217606 108753
217608 108752
217610 108751
217612 108750
217614 108749
217616 108748
217618 108747
217620 108746
217622 108745
217624 108744
217626 108743
217628 108742
326349 108763
435112 108763
436598 699561
437777 538257
445970 821076
451239 945741
457634 872722
462812 747486
468703 671675
482556 601085
483987 646948
493525 987970
499967 954764
500527 879438
509158 903243
510368 658269
511276 569529
515454 710616
520036 668048
529746 807285
530401 626964
531284 929571
531783 799919
536284 548721
537284 608327
537899 536863
640640 102738
640642 102737
640644 102736
640646 102735
640648 102734
640650 102733
640652 102732
640654 102731
640656 102730
640658 102729
640660 102728
640662 102727
640664 102726
640666 102725
640668 102724
640670 102723
640672 102722
640674 102721
640676 102720
640678 102719
640680 102718
640682 102717
640684 102716
640686 102715
743401
23
10 53
11 63
12 74
13 13
14 97
15 31
16 22
17 57
18 10
19 17
20 16
30 10
40 10
41 23
42 91
43 96
44 86
45 88
46 84
47 86
48 19
49 15
50 27
60
99
458898 1000000
917702 458801
917704 458800
917706 458799
917708 458798
917710 458797
917712 458796
917714 458795
917716 458794
917718 458793
917720 458792
917722 458791
917724 458790
917726 458789
917728 458788
917730 458787
917732 458786
917734 458785
917736 458784
917738 458783
917740 458782
917742 458781
917744 458780
917746 458779
917748 458778
917750 458777
917752 458776
917754 458775
917756 458774
917758 458773
917760 458772
917762 458771
917764 458770
917766 458769
917768 458768
917770 458767
917772 458766
917774 458765
917776 458764
917778 458763
917780 458762
917782 458761
917784 458760
917786 458759
917788 458758
917790 458757
917792 458756
917794 458755
917796 458754
1376550 458753
1835304 458754
2293964 458657
2293966 458656
2293968 458655
2293970 458654
2293972 458653
2293974 458652
2293976 458651
2293978 458650
2293980 458649
2293982 458648
2293984 458647
2293986 458646
2293988 458645
2293990 458644
2293992 458643
2293994 458642
2293996 458641
2293998 458640
2294000 458639
2294002 458638
2294004 458637
2294006 458636
2294008 458635
2294010 458634
2294012 458633
2294014 458632
2294016 458631
2294018 458630
2294020 458629
2294022 458628
2294024 458627
2294026 458626
2294028 458625
2294030 458624
2294032 458623
2294034 458622
2294036 458621
2294038 458620
2294040 458619
2294042 458618
2294044 458617
2294046 458616
2294048 458615
2294050 458614
2294052 458613
2294054 458612
2294056 458611
2294058 458610
2752668
2
10 10
11 10
20
99
10 92
11 83
12 24
15 92
16 85
17 43
18 20
19 14
20 89
30 10
40 10
42 42
43 13
44 94
46 37
47 92
48 83
49 82
50 22
60 10
70 10
72 51
74 29
75 67
76 18
77 73
78 81
79 66
80 77
90 10
100 10
101 44
102 66
103 52
104 11
105 22
107 69
109 79
110 60
120 10
130 10
131 12
132 74
133 22
134 32
136 75
137 96
138 69
139 16
140 10
150 10
151 34
152 90
154 54
155 75
157 57
158 83
159 66
160 21
170 10
180 10
181 55
183 68
184 45
185 56
186 17
187 29
189 57
190 55
200 10
210 10
211 29
212 73
214 18
215 33
216 76
217 86
219 31
220 89
230 10
240 10
241 78
242 99
244 22
246 57
247 83
248 70
249 35
250 66
260 10
270 10
271 16
272 43
274 15
275 90
276 33
277 58
278 84
279 85
288
99
255416 1000000
265882 903816
358453 762779
395309 571331
456945 571590
658468 201520
658470 201519
658472 201518
658474 201517
859990 201518
1061508 201518
1168738 726761
1192815 704754
1247978 646134
1259143 710469
1263020 201509
1263022 201508
1263024 201507
1263026 201506
1464529 201509
1666038 201509
1674729 748263
1737723 796810
1833901 756204
1852904 737906
2039764 186857
2039766 186856
2039768 186855
2039770 186854
2226624 186854
2413478 186854
2492331 526292
2513378 677118
2534183 764278
2567344 767464
2721204 153857
2721206 153856
2721208 153855
2721210 153854
2875063 153855
3028918 153855
3062373 880631
3119677 668755
3140087 541651
3157404 641438
3182767 153846
3182769 153845
3182771 153844
3182773 153843
3336616 153843
3490459 153843
3574733 562088
3607747 704435
3612837 991927
3625392 740177
3644296 153834
3644298 153833
3644300 153832
3644302 153831
3798130 153834
3951964 153834
3955415 970634
3974729 883195
4090595 518491
4095609 602925
4105792 153825
4105794 153824
4105796 153823
4105798 153822
4259618 153824
4413442 153824
4451706 592773
4478885 866519
4500696 879922
4535657 721105
4657866 122206
4657868 122205
4657870 122204
4657872 122203
4780074 122204
4902278 122204
4933118 776066
4939735 548869
4949875 572357
4985727 567982
5069170 83440
5069172 83439
5069174 83438
5069176 83437
5152613 83437
5236050 83437
5287647 580108
5303379 823223
5307090 727854
5318915 536579
5319481 83428
5319483 83427
5319485 83426
5319487 83425
5402912
95
615222 1000000
693307 529759
712059 841700
728049 745629
764893 962437
817518 884611
889784 954175
908219 970509
996207 887757
1065695 827580
1069282 665162
1103051 606157
1142736 999068
1157001 639862
1170408 702075
1228509 971770
1841768 613256
1841770 613255
1841772 613254
1841774 613253
1841776 613252
1841778 613251
1841780 613250
1841782 613249
1841784 613248
1841786 613247
1841788 613246
1841790 613245
1841792 613244
1841794 613243
1841796 613242
2455032 613247
3068280 613248
3233283 812127
3297695 620393
3332283 824869
3359383 656824
3399660 693701
3411484 581302
3417540 785862
3446794 979657
3462799 852452
3476380 848594
3499256 689253
3529010 692812
3655422 950206
3659057 923529
3661773 562202
4249806 590746
4249808 590745
4249810 590744
4249812 590743
4249814 590742
4249816 590741
4249818 590740
4249820 590739
4249822 590738
4249824 590737
4249826 590736
4249828 590735
4249830 590734
4249832 590733
4249834 590732
4840566 590732
5431298 590732
5442777 849856
5470287 923966
5489278 978275
5535445 647877
5548137 565500
5559963 975982
5575411 690672
5622279 735573
5778097 981480
5798102 680205
5817553 641105
5854493 968899
5864551 520309
5922516 694816
5996094 586643
6560862 564765
6560864 564764
6560866 564763
6560868 564762
6560870 564761
6560872 564760
6560874 564759
6560876 564758
6560878 564757
6560880 564756
6560882 564755
6560884 564754
6560886 564753
6560888 564752
6560890 564751
7125641
7
10 10
15 5
20 9
25 5
30 10
35 5
40 10
49
99
393386 1000000
786768 393379
786770 393378
786772 393377
1180148 393378
1573526 393378
1966900 393371
1966902 393370
1966904 393369
2360271 393371
2753642 393371
3147009 393364
3147011 393363
3147013 393362
3540374 393363
3933737 393363
4327096 393356
4327098 393355
4327100 393354
4720454 393354
5113808 393354
5507158 393347
5507160 393346
5507162 393345
5900506 393346
6293852 393346
6687194 393339
6687196 393338
6687198 393337
7080533 393338
7473872 393339
7867207 393332
7867209 393331
7867211 393330
8260541 393330
8653871 393330
9047197 393323
9047199 393322
9047201 393321
9440522 393321
9833843 393321
10227160 393314
10227162 393313
10227164 393312
10620475 393313
11013788 393313
11407097 393306
11407099 393305
11407101 393304
11800405 393304
12193709 393304
12587009 393297
12587011 393296
12587013 393295
12980308 393295
13373603 393295
13766894 393288
13766896 393287
13766898 393286
14160184 393286
14553470 393286
14946752 393279
14946754 393278
14946756 393277
15340033 393277
15733310 393277
16126583 393270
16126585 393269
16126587 393268
16519855 393268
16913123 393268
17306387 393261
17306389 393260
17306391 393259
17699650 393259
18092909 393259
18486164 393252
18486166 393251
18486168 393250
18879418 393250
19272668 393250
19665914 393243
19665916 393242
19665918 393241
20059157 393243
20452400 393243
20845639 393236
20845641 393235
20845643 393234
21238877 393234
21632111 393234
22025341 393227
22025343 393226
22025345 393225
22418569 393226
22811795 393226
23205017 393219
23205019 393218
23205021 393217
23598238
4
5 5
8 3
9 4
11 3
14
2
10 1000
100 10
110
99
588591 1000000
1177178 588584
1177180 588583
1177182 588582
1765762 588584
2354346 588584
2942926 588577
2942928 588576
2942930 588575
3531503 588577
4120080 588577
4708653 588570
4708655 588569
4708657 588568
5297223 588570
5885793 588570
6474359 588563
6474361 588562
6474363 588561
7062923 588562
7651485 588562
8240043 588555
8240045 588554
8240047 588553
8828598 588555
9417153 588555
10005704 588548
10005706 588547
10005708 588546
10594253 588547
11182800 588547
11771343 588540
11771345 588539
11771347 588538
12359885 588538
12948423 588538
13536957 588531
13536959 588530
13536961 588529
14125490 588529
14714019 588529
15302544 588522
15302546 588521
15302548 588520
15891067 588521
16479588 588521
17068105 588514
17068107 588513
17068109 588512
17656619 588514
18245133 588514
18833643 588507
18833645 588506
18833647 588505
19422150 588507
20010657 588507
20599160 588500
20599162 588499
20599164 588498
21187661 588499
21776160 588499
22364655 588492
22364657 588491
22364659 588490
22953147 588492
23541639 588492
24130127 588485
24130129 588484
24130131 588483
24718613 588484
25307097 588484
25895577 588477
25895579 588476
25895581 588475
26484054 588477
27072531 588477
27661004 588470
27661006 588469
27661008 588468
28249474 588470
28837944 588470
29426410 588463
29426412 588462
29426414 588461
30014874 588462
30603336 588462
31191794 588455
31191796 588454
31191798 588453
31780249 588455
32368704 588455
32957155 588448
32957157 588447
32957159 588446
33545604 588447
34134051 588447
34722494 588440
34722496 588439
34722498 588438
35310936
