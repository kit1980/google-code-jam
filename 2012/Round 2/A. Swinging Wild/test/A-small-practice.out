Case #1: YES
Case #2: NO
Case #3: YES
Case #4: NO
Case #5: NO
Case #6: NO
Case #7: YES
Case #8: YES
Case #9: NO
Case #10: YES
Case #11: YES
Case #12: NO
Case #13: NO
Case #14: YES
Case #15: YES
Case #16: NO
Case #17: YES
Case #18: YES
Case #19: YES
Case #20: YES
Case #21: NO
Case #22: YES
Case #23: YES
Case #24: YES
Case #25: NO
Case #26: NO
Case #27: NO
Case #28: YES
Case #29: NO
Case #30: YES
